example/config.yaml

```diff
@@ -18,12 +18,14 @@ settings:
     # configured Juniper equipment reports data over the JTI stream.
     interval: 1s
   write:
-    enabled: false
+    disable: true
   listen:
-    enabled: false
+    disable: true
   cache:
     enabled: true
     ttl: 5m
 dynamicRegistration:
   config:
-    - address: udp://0.0.0.0:5566
\ No newline at end of file
+    - address: udp://0.0.0.0:5566
+      context:
```

Comment:
:+1: Was looking for where this would show up. nice to see its pretty front and center.

Reply:
(: